packages/echo/src/client.ts

```diff
@@ -626,11 +633,16 @@ export class Echo {
   }
 
   private createExecutionInputs(event: IEvent, workflow: DiscoverWorkflowOutput): Record<string, unknown> {
-    const executionData = event.data;
+    let payload = event.data;
+    if (event.action === 'preview') {
+      const mockResult = this.mock(workflow.data.schema);
+
+      payload = Object.assign(mockResult, payload);
```

Comment:
Does this suggestion work? I'm not sure I understand why `Object.assign` is used
```suggestion
      payload = this.mock(workflow.data.schema);
```

Reply:
The reason is that I want to polyfill all the values except the ones the user has provided in the inputs field. So what we need is merging the user provided inputs and the mocked object